From e157bebed4bc7ee45441d2ac34acf97786f416c5 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Wed, 20 Jun 2018 10:22:32 +0530
Subject: [PATCH 442/448] staging: fsl_ppfe/eth: reorganize pfe_netdev_ops

Reorganize members of struct pfe_netdev_ops to match with the order
of members in struct net_device_ops defined in include/linux/netdevice.h

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 drivers/staging/fsl_ppfe/pfe_eth.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fsl_ppfe/pfe_eth.c b/drivers/staging/fsl_ppfe/pfe_eth.c
index f933f3a..6fa8a3c 100644
--- a/drivers/staging/fsl_ppfe/pfe_eth.c
+++ b/drivers/staging/fsl_ppfe/pfe_eth.c
@@ -2242,11 +2242,11 @@ static int pfe_eth_high_poll(struct napi_struct *napi, int budget)
 	.ndo_stop = pfe_eth_close,
 	.ndo_start_xmit = pfe_eth_send_packet,
 	.ndo_select_queue = pfe_eth_select_queue,
-	.ndo_get_stats = pfe_eth_get_stats,
-	.ndo_set_mac_address = pfe_eth_set_mac_address,
 	.ndo_set_rx_mode = pfe_eth_set_multi,
-	.ndo_set_features = pfe_eth_set_features,
+	.ndo_set_mac_address = pfe_eth_set_mac_address,
 	.ndo_validate_addr = eth_validate_addr,
+	.ndo_get_stats = pfe_eth_get_stats,
+	.ndo_set_features = pfe_eth_set_features,
 };
 
 /* pfe_eth_init_one
-- 
1.7.1

